spec/lookup_spec.rb

```diff
@@ -32,24 +30,28 @@
           :key_conditions=>{
             "component_key"=> {
               :comparison_operator=>"EQ",
-              :attribute_value_list=>[{"s"=>"id/9dd916afd5516828a91d259967fd394a"}]
+              :attribute_value_list=>[{"s"=>"id/218c835cec343537589dbf1619532e4d"}]
```

Comment:
I swapped the order of the params around, with the options and the batch_version in these tests, as I noticed the instance being tested has them the other way around.